features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -2028,6 +2028,22 @@
             <Permissions>/permission/admin/manage/identity/organizationmgt/update</Permissions>
             <Scopes>internal_organization_update</Scopes>
         </Resource>
+        <Resource context="(.*)/api/server/v1/guests/invite" secured="true" http-method="POST">
+            <Permissions>/permission/admin/manage/identity/guestmgt/user/invite/add</Permissions>
+            <Scopes>internal_guest_mgt_invite_add</Scopes>
```

Comment:
Shall we use created instead of add

Reply:
Then we need to introduce the create scope. Currently we have only these.
internal_guest_mgt_invite_add 
internal_guest_mgt_invite_delete 
internal_guest_mgt_invite_list 
internal_guest_mgt_invite_update 
internal_guest_mgt_invite_validate